src/diffusers/quantizers/bitsandbytes/utils.py

```diff
@@ -304,3 +318,80 @@ def _check_bnb_status(module) -> Union[bool, bool]:
         and getattr(module, "quantization_method", None) == QuantizationMethod.BITS_AND_BYTES
     )
     return is_loaded_in_4bit_bnb or is_loaded_in_8bit_bnb, is_loaded_in_4bit_bnb, is_loaded_in_8bit_bnb
+
+
+def _validate_bnb_multi_backend_availability(raise_exception):
```

Comment:
@matthewdouglas I wonder if it makes sense have these as utility functions in `bitsandbytes` so that they can be reused in `transformers` and `diffusers` (and any other libraries)?